Makefile

```diff
@@ -6,10 +6,10 @@
 VERSION ?= 0.0.1
 
 # CHANNELS define the bundle channels used in the bundle.
-# Add a new line here if you would like to change its default config. (E.g CHANNELS = "preview,fast,stable")
+# Add a new line here if you would like to change its default config. (E.g CHANNELS = "fast,stable")
```

Comment:
Not necessary to change this file, this is for OLM I believe for the bundles there.

Reply:
thanks, I rolled this change back